DCCpp_Uno/src/PacketRegister.cpp

```diff
@@ -324,38 +327,37 @@ void RegisterList::writeCVBit(const char *s) volatile{
 
   if(sscanf(s,"%d %d %d %d %d",&cv,&bNum,&bValue,&callBack,&callBackSub)!=5)          // cv = 1-1024
     return;
-  cv--;                              // actual CV addresses are cv-1 (0-1023)
+  cv--;                                 // actual CV addresses are cv-1 (0-1023)
   bValue=bValue%2;
   bNum=bNum%8;
 
   bWrite[0]=0x78+(highByte(cv)&0x03);   // any CV>1023 will become modulus(1024) due to bit-mask of 0x03
   bWrite[1]=lowByte(cv);
   bWrite[2]=0xF0+bValue*8+bNum;
-
-  loadPacket(0,resetPacket,2,1);
-  loadPacket(0,bWrite,3,4);
-  loadPacket(0,resetPacket,2,1);
-  loadPacket(0,idlePacket,2,10);
-
+  loadPacket(0,resetPacket,2,3);        // NMRA recommends starting with 3 reset packets
+  loadPacket(0,bWrite,3,5);             // NMRA recommends 5 verify packets
+  loadPacket(0,bWtite,3,6);             // NMRA recommends 6 write or reset packets for decoder recovery time
+      
   c=0;
   d=0;
   base=0;
 
   for(int j=0;j<ACK_BASE_COUNT;j++)
     base+=analogRead(CURRENT_MONITOR_PIN_PROG);
   base/=ACK_BASE_COUNT;
-
+  
   bitClear(bWrite[2],4);              // change instruction code from Write Bit to Verify Bit
-
-  loadPacket(0,resetPacket,2,3);          // NMRA recommends starting with 3 reset packets
-  loadPacket(0,bWrite,3,5);               // NMRA recommends 5 verfy packets
-  loadPacket(0,resetPacket,2,1);          // forces code to wait until all repeats of bRead are completed (and decoder begins to respond)
-
+  loadPacket(0,resetPacket,2,3);      // NMRA recommends starting with 3 reset packets
+  loadPacket(0,bWrite,3,5);           // NMRA recommends 5 verify packets
+  loadPacket(0,bWtite,3,6);           // NMRA recommends 6 write or reset packets for decoder recovery time
```

Comment:
typo here as well

Reply:
What is proper etiquette for making changes like this? leave it up to the original author of the change to fix, go into his branch, fix it and commit it (is a new PR required?), or create your own branch from the authors, do the fix and issue a PR?

Reply:
Fix it in the branch before merge. Once the actions are in place we will also be able to catch these sort of typos a lot easier.